js/load-scene.js

```diff
@@ -63,6 +63,7 @@ AFRAME.registerComponent('loadscene', {
 
 
     document.querySelector('a-scene').addEventListener('loaded', function () {
+      document.getElementById('get-ready').style.display = 'none';
```

Comment:
why do we hide this on `'loaded'`? I think it should stay visible. and when the user exits and re-enters VR, the Get Ready dialog should appear again.

Reply:
The actual instructions are in the A-Frame scene itself. This displays before A-Frame is fully rendering. You shouldn't see this unless you are fresh to the page only.  This was done for the sake of consistency and maintaining what behavior was already there. It's a "Loading" page basically.

Reply:
ah, okay - is that the expected behaviour? I assumed the instructions were supposed to appear once you enter VR. can you confirm, @lschatzkin?

either way, can that be handled in a separate issue and PR?

Reply:
They are displayed on load when you are in VR, or just right of the cursor if you're viewing on the 2D browser.

Reply:
I didn't see it displayed when I enter VR anymore in this PR

Reply:
<img width="247" alt="image" src="https://user-images.githubusercontent.com/1560884/51714948-ba9ae600-1feb-11e9-9bc6-6e4c4e95ee89.png">

You might not be oriented in the right direction when you enter VR? It's definitely there.

Reply:
> ah, okay - is that the expected behaviour? I assumed the instructions were supposed to appear once you enter VR. can you confirm, @lschatzkin?
> 
> either way, can that be handled in a separate issue and PR?
It'd be great if the intro/instructions appears when the user enters VR when that option is available (i.e. not mobile or desktop browser).

Reply:
heh, weird. you're right. testing this branch right now, I can see the **Get Started** modal as expected.

nothing to change here.

I did file a feature request, enhancement issue #85 to dismiss the **Get Started** modal when the player starts playing the game. we can continue the UX discussion in that issue.